Rakefile

```diff
@@ -0,0 +1,6 @@
+# Add your own tasks in files placed in lib/tasks ending in .rake,
```

Comment:
Whats this file about?

Reply:
It's a file for writing Rake tasks in Ruby and allowing you to execute them from the command line.

Reply:
What is "Rake Task"?

Reply:
Basically anything that can be scripted in ruby. For e.g. it could be a task to print to the console, concatenate files, run tests, etc.

Reply:
Completely incorrect

Reply:
a rake task is basically a script that you can run using the rake command.